### PR TITLE
Remove widths to match page default padding

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?

Remove widths to match page default padding

#### What problem is this solving?

Width definitions were overriding padding definition.

#### How should this be manually tested?

[Access the workspace](https://padding--storecomponents.myvtex.com/) and resize the screen.

#### Screenshots or example usage
image
image

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__tests__/CategoryMenu.test.js [hunk 1]
```diff
@@ -52,10 +53,6 @@ describe('CategoryMenu component', () => {
     expect(wrapper).toBeDefined()
   })
 
-  it('should match snapshot', () => {
-    expect(wrapper).toMatchSnapshot()
```
Comment (lucasecdb) [inline react/__tests__/CategoryMenu.test.js]: Why did you remove this?

File: react/index.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
-import React, { Component, Fragment } from 'react'
+import './global.css'
```
Comment (lucasecdb) [inline react/index.js]: Relative imports should come after the absolute ones. :pick:

Comment (estacioneto) [reply react/index.js]: You sure? Typescript Hero organizes it that way. I didn't see any pattern about the imports order besides the natural language one.

Comment (lucasecdb) [reply react/index.js]: yep, it's what we are adopting

Comment (estacioneto) [reply react/index.js]: It organizes the "full" imports first. Like this CSS import that has no attribution. The relative imports (the ones with "attribution") come right after. Then, we have the absolute imports.

Comment (lucasecdb) [reply react/index.js]: the way the other apps are structured, absolute imports come first, then the relative "with attribution", then the ones without it, like this CSS one.

Comment (estacioneto) [reply react/index.js]: Another cool extension to disable. Thank you :)

Comment (estacioneto) [reply react/index.js]: Good to know that has no use of alphabetic order in the other apps too. Gr8 m8
